packages/host/src/Host.tsx

```diff
@@ -60,45 +60,47 @@ const Host: FC = () => {
           <AppThemeProvider>
             <BrowserRouter>
               <Viewport>
+                <AppBar />
```

Comment:
not necessary to move it now - I was thinking to change all of these to flex layout.
Kept this as a fixed position so that it is always on the top of the page and as such, doesn't matter where it is in the dom